.travis.yml

```diff
@@ -2,3 +2,5 @@ language: node_js
 
 node_js:
   - '6'
+  - '4'
+  - '0.10'
```

Comment:
I see... Well, if you think you can make the CI pass through v0.10, feel free to open another PR :smiley: